docs/advanced-features/loop-rendering.md

```
@@ -10,7 +10,7 @@
    ![addVar-17](./imgs/addVar-17.png)
 
 3. 选中组件，在组件属性设置面板选择“高级”。
-4. 单击“循环数据“后的![](./imgs/icon-code.png)，进行变量绑定。
+4. 单击“循环数据“后的![](./imgs/icon-code.png)<!-- {.image-inline} -->，进行变量绑定。
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)